ticket_booking_app/src/main/java/com/mordor/Utils/ResourceNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.mordor.Utils;
+
+public class ResourceNotFoundException extends Exception{

```

Comment:
Przez to, że klasa ta dziedziczy po Exception, musisz robić w kodzie konstrukcje try-catch i throws. Wiesz jak można to zmienić (żeby nadal byl exception, tylko bez wymuszania kodu do jego obsługi)?